packages/console/src/pages/SignInExperience/tabs/SignUpAndSignIn/AdvancedOptions.tsx

```diff
@@ -17,6 +18,19 @@ function AdvancedOptions() {
   return (
     <Card>
       <div className={styles.title}>{t('title')}</div>
+      <FormField title="sign_in_exp.sign_up_and_sign_in.advanced_options.enable_single_sign_on">
+        <Switch
+          {...register('singleSignOnEnabled')}
+          label={t('enable_single_sign_on_description')}
```

Comment:
If there is no sso connectors, the use can still enable this switch?

Reply:
yes, this is by design.